etl/steps/data/garden/irena/2023-12-12/renewable_electricity_capacity.meta.yml

```diff
@@ -1,121 +1,59 @@
-# TODO: This file is a duplicate of the previous step. It is not yet used in the dag and should be updated soon.
+definitions:
+  common:
+    presentation:
+      topic_tags:
+        - Energy
+    short_unit: MW
+    unit: megawatts
+    display:
+      numDecimalPlaces: 0
+    description_short: |-
+      Measured in megawatts.
+
+dataset:
+  update_period_days: 365
 
 tables:
   renewable_electricity_capacity:
+    title: Renewable electricity capacity
     variables:
       bagasse:
         title: Bagasse capacity
-        short_unit: MW
-        unit: megawatts
-        description: |
-          Bagasse is a byproduct from the process of sugar cane refining, often used as a biofuel for the production of heat, energy, and electricity.
-        display:
-          numDecimalPlaces: 0
+        description_key:
+        - Bagasse is a byproduct from the process of sugar cane refining, often used as a biofuel for the production of heat, energy, and electricity.
       bioenergy:
         title: Bioenergy capacity
-        short_unit: MW
-        unit: megawatts
-        display:
-          numDecimalPlaces: 0
       biogas:
         title: Biogas capacity
-        short_unit: MW
-        unit: megawatts
-        display:
-          numDecimalPlaces: 0
       concentrated_solar_power:
         title: Concentrated solar power capacity
-        short_unit: MW
-        unit: megawatts
-        display:
-          numDecimalPlaces: 0
       geothermal_energy:
         title: Geothermal energy capacity
-        short_unit: MW
-        unit: megawatts
-        display:
-          numDecimalPlaces: 0
       hydropower:
         title: Hydropower capacity
-        short_unit: MW
-        unit: megawatts
-        display:
-          numDecimalPlaces: 0
       liquid_biofuels:
         title: Liquid biofuels capacity
-        short_unit: MW
-        unit: megawatts
-        display:
-          numDecimalPlaces: 0
       marine_energy:
         title: Marine energy capacity
```

Comment:
it'd be useful to include similar description_key as you do for Bagasse for all/most of these. It's not immediately clear what `Marine energy capacity `or `Liquid biofuels capacity mean`.  Spelling out what capacity means would also be nice - ie. `Amount of electricity or energy that can be generated from x`.